test/e2e-v2/script/env

```diff
@@ -23,6 +23,6 @@ SW_AGENT_CLIENT_JS_COMMIT=af0565a67d382b683c1dbd94c379b7080db61449
 SW_AGENT_CLIENT_JS_TEST_COMMIT=4f1eb1dcdbde3ec4a38534bf01dded4ab5d2f016
 SW_KUBERNETES_COMMIT_SHA=b670c41d94a82ddefcf466d54bab5c492d88d772
 SW_ROVER_COMMIT=8550199e98c9f5a4b2058878a0a899ffb73fe461
-SW_BANYANDB_COMMIT=1cc0a42bc41dca9b8f4e1b3c4fd13b283f24f817
+SW_BANYANDB_COMMIT=adbd3e87df7f84e5d1904fcf40476d2e81842058
```

Comment:
This is the commit ID for 0.3.1. Have cross-checked the release tag https://github.com/apache/skywalking-banyandb/tree/v0.3.1